agent/local/state.go

```diff
@@ -879,6 +881,7 @@ func (l *State) deleteCheck(id types.CheckID) error {
 		// todo(fs): some backoff strategy might be a better solution
 		l.checks[id].InSync = true
 		l.logger.Printf("[WARN] agent: Check %q deregistration blocked by ACLs", id)
+		metrics.IncrCounterWithLabels([]string{"consul", "acl", "blocked", "check", "deregistration"}, 1, []metrics.Label{{Name: "id", Value: string(id)}})
```

Comment:
You don't need to add "consul" at the beginning of the series name - that gets added automatically. Same for the rest of the metrics being added.

Reply:
DONE

Reply:
DONE